### ISSUE TITLE
Breaking Upstream Change With Logger

### ISSUE THREAD

**Issue by bliu13** [AriPerkkio/vitest-sonar-reporter#15]
There is a breaking change introduced in [v0.18.0](https://github.com/vitest-dev/vitest/releases/tag/v0.18.0) where `ctx.log` was changed to `ctx.logger.log`.

**Comment by AriPerkkio** [comment]
Thanks for the report @bliu13. I'm able to reproduce this with `vitest@0.18.0`. 

```
⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯ Unhandled Error ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
TypeError: this.ctx.log is not a function
 ❯ SonarReporter.onFinished test/vite.test-config.ts.js?t=1657562744345:118:14
 ❯ node_modules/vitest/dist/chunk-vite-node-externalize.0791f2ed.mjs:10705:51
 ❯ Vitest.report node_modules/vitest/dist/chunk-vite-node-externalize.0791f2ed.mjs:10703:38
 ❯ node_modules/vitest/dist/chunk-vite-node-externalize.0791f2ed.mjs:10551:18
 ❯ processTicksAndRejections node:internal/process/task_queues:96:5
 ❯ Vitest.runFiles node_modules/vitest/dist/chunk-vite-node-externalize.0791f2ed.mjs:10557:12
 ❯ Vitest.start node_modules/vitest/dist/chunk-vite-node-externalize.0791f2ed.mjs:10478:5
 ❯ startVitest node_modules/vitest/dist/chunk-vite-node-externalize.0791f2ed.mjs:11203:5
 ❯ CAC.start node_modules/vitest/dist/cli.mjs:666:9
```

Fix will be landing soon. 😎

**Comment by AriPerkkio** [comment]
This is now fixed and released in `0.3.0`.
